docs/source/new-docs/getting-started/installing/index.rst

```diff
@@ -4,3 +4,228 @@
+or Docker installation methods.
+
+Get Conda
+^^^^^^^^^
+
+- Download the appropriate version of `Conda
+  <https://www.anaconda.com/download#downloads>`_,  `Miniconda
+  <https://docs.conda.io/en/latest/miniconda.html>`_, or
+  `Miniforge/Mambaforge <https://github.com/conda-forge/miniforge#download>`_.
+
+  For example, for Linux with Intel chips, one of the following might work:
+
+  .. code:: bash
+
+      wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
+
+  To find your specific version, visit the `Miniforge/Mambaforge github page
+  <https://github.com/conda-forge/miniforge/releases/>`_.
+
+- Make the install script executable and run the installer,
+  following the prompts. Eg.:
+
+  .. code:: bash
+
+      chmod u+x ./Mambaforge-Linux-x86_64.sh
+      # After installing, follow instructions regarding conda init / restarting your terminal !
+      # Make sure you set env vars again if you restart your terminal !
+      ./Mambaforge-Linux-x86_64.sh
+
+  .. code:: bash
+
+      rm -f ./Mambaforge-Linux-x86_64.sh
+
+Create and activate a conda environment with dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Next we'll create a conda environment named ``geoips`` that contains all system
+requirements for GeoIPS.
+
+.. code:: bash
+
+    mamba create -y -n geoips -c conda-forge python=3.10 openblas git
+
+.. note::
+
+    If you would like to run plugins that require fortran, you will need to install ``gfortran``.
+
+.. warning::
+
+        If you are using an ARM machine, you may need to install ``pyhdf`` with conda.
+        If you run into ``fatal error: 'hdf.h' file not found`` when installing ``pyhdf``,
+        you may need to install ``pyhdf`` with conda. You can do this with the following command:
+
+        .. code:: bash
+
+            mamba create -y -n geoips -c conda-forge python=3.10 openblas git **pyhdf**
+
+Activate the conda environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You will need to run ``mamba activate geoips`` (or ``conda activate geoips``)
+every time you want to run or work on GeoIPS. Eg.
+
+.. code:: bash
+
+    mamba activate geoips
+
+Install system dependencies directly
+------------------------------------
+
+If you are an expert user, you may want to install the dependencies manually. Do not do this if you already installed
+dependencies with Conda.
+
+Required
+^^^^^^^^
+
+* ``git``
+* ``openblas`` (required for scipy pip install)
+* ``make`` (required for pypublicdecompwt)
+* ``python`` >= 3.9
+
+Optional
+^^^^^^^^
+
+* ``gfortran`` (required for plugins including fortran builds)
+* ``gcc`` and ``g++`` (required for plugins including fortran or C builds, and ARM machines)
+* ``pdflatex`` (optional, for building pdf documentation)
+* ``wget`` (required for downloading test data)
+* ``make`` (optional, for building packages on ARM machines)
+* Test data repos can be installed in `$GEOIPS_TESTDATA_DIR`
```

Comment:
I still think this should not go in the standard installation.  We should keep the standard installation very clean and vanilla, to avoid unnecessary confusion.  Our standard supported installation is conda - and this is not necessary if you are installing with conda.

Reply:
To echo Mindy, I also agree that the standard installation instructions (conda install on a Linux machine) should be clean and as minimal as possible. I think it not only makes it easier for the end-user, but also makes it easier for us to help troubleshoot a standard installation if needed.

Reply:
Great! I changed this document to be conda only 🙂